openlibrary/macros/daisy.html

```diff
@@ -1,19 +1,18 @@
-$def with(page, url='', msg='', block_name=None)
+$def with(page, url='', enc=False, block_name=None)
```

Comment:
I think dropping `msg` breaks the search page.

Reply:
Thank you, this is a BLOCKER!

Reply:
done